src/Termonad/Types.hs

```diff
@@ -455,6 +461,7 @@ defaultConfigOptions =
     , showTabBar = ShowTabBarIfNeeded
     , cursorBlinkMode = CursorBlinkModeOn
     , boldIsBright = False
+    , enableSixel = False
```

Comment:
@junjihashimoto I changed this to `False` because <https://gitlab.gnome.org/GNOME/vte/-/issues/253#note_1491229> said

> And anyway, I don't want to have SIXEL enabled in a distro release. The SIXEL code still has important and fundamental problems unresolved.

Although I'd be happy to accept a PR that defaults SIXEL support to `True` in Termonad once the upstream VTE maintainers are more confident in the code.

Reply:
Maybe add a comment saying that setting it to True doesn't currently do anything?

Reply:
@gelisam Thanks for taking a look at this.

I didn't actually read through all of [https://gitlab.gnome.org/GNOME/vte/-/issues/253](https://gitlab.gnome.org/GNOME/vte/-/issues/253#note_1491229), but it seemed to me that the gist of the last few messages is that if you have a new enough version of VTE, SIXEL support does work to some extent (at least partially?).

Although Junji's message does make it seem like it doesn't work at all.  I'm not sure exactly what wording makes the most sense here.

Reply:
I agree that the thread makes it clear that displaying images should work already, with the "slight" caveat that their memory is never released. I volunteer to try compiling libvte with sixel support enabled in order to double-check that indeed, with the proper version of libvte, enabling sixel does do something.

Reply:
@gelisam Thanks!

I'll go ahead and release this, and based on the result of your tests, we can change the wording here.

Reply:
I see you have wisely learned not to wait for me to finish things!

Reply:
Oh, haha, I was just thinking I would probably be the one to forget to actually merge this!

If your tests come back and it turns out nothing seems to work, we can always just release updated haddocks and potentially deprecate 4.3.0.0 version I just uploaded.

Reply:
I am happy to confirm that with the right vte library, termonad now does support sixtel!

![termonad-sixel-screenshot](https://user-images.githubusercontent.com/49000/196088769-69feafec-73c1-4432-9ad7-93a31b3710a9.png)

To help others reproduce my success, here is what I did.

```bash
$ git clone git@github.com:GNOME/vte.git
$ cd vte
$ meson -Dsixel=true _build --prefix=~/custom
$ ninja -C _build
$ ninja -C _build install
$ export PKG_CONFIG_PATH="$HOME/custom/lib64/pkgconfig:$PKG_CONFIG_PATH"
$ export LD_LIBRARY_PATH="$HOME/custom/lib64"
$ cd -

# edit ~/.config/termonad/termonad.hs to set enableSixel to True

$ git clone git@github.com:cdepillabout/termonad.git
$ cd termonad
$ stack build
$ stack exec -- termonad

# in the new termonad window
$ wget "https://raw.githubusercontent.com/saitoha/libsixel/master/images/snake.six"
$ cat snake.six
# behold! a snake picture!
```

Reply:
It would probably be a good idea to figure out how to do this without `LD_LIBRARY_PATH`.

Reply:
@gelisam Thanks!

I pushed a commit directly to `master` that sort of updates the `CHANGELOG` and haddocks with the results of your tests: https://github.com/cdepillabout/termonad/pull/219#discussion_r996602017

Reply:
> It would probably be a good idea to figure out how to do this without `LD_LIBRARY_PATH`.

Turns out adding the following to `stack.yaml` did the trick, but only after I blew up my `~/.stack` in order to force a recompilation of `gi-vte`.

```
extra-lib-dirs:
- /home/gelisam/custom/lib64
```